scripts/remove-idle-users.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import sys
+import json
+import urllib
+import requests
+import re
+
+## debug request
+import httplib as http_client
+http_client.HTTPConnection.debuglevel = 1
+
+def get_room_id(homeserver, alias, token):
+    res = requests.get(homeserver + "/_matrix/client/r0/directory/room/" + urllib.quote(alias) + "?access_token=" + token).json()
+    return res.get("room_id", None)
```

Comment:
Nit: Rather than return a nullable `room_id`, I would just do:
```py
res = requests.get(homeserver + "/_matrix/client/r0/directory/room/" + urllib.quote(alias) + "?access_token=" + token)
res.raise_for_status()
return res.json()["room_id"]
```

This will throw on non-2xx and if `room_id` doesn't exist, both of which you want to do. You can then remove the null guard on `:94`.